File: scripts/python/factorial.py
#Factorail of a Number
n=input("Enter n Value")
fact=1
if n==0 :
	print("The Factorail of %d is 1"%(n))
else :
	for i in range(1,n+1) :
		fact=fact*i
	print("The Factorail of %d is %d "%(n,fact))

File: ruby/factorial.rb
#Factorail of a Number
puts "Enter a Number"
n=gets
fact=1
if n==0
  puts "The Factorail of #{n} is 1"
else
  for i in 1..Integer(n) do
    fact=fact*i
  end
puts "The Factorail of #{n} is #{fact} "
end
